lib/simp/beaker_helpers.rb

```diff
@@ -2,7 +2,7 @@ module Simp; end
 
 module Simp::BeakerHelpers
   require 'simp/beaker_helpers/version'
-  DEFAULT_PUPPET_AGENT_VERSION = '1.8.3'
+  DEFAULT_PUPPET_AGENT_VERSION = '1.10.2'
```

Comment:
@jeefberkey there's a fugly bit in the install_puppet function that gives PUPPET_VERSION hierarchy over this default. PUPPET_INSTALL_VERSION > PUPPET_VERSION > DEFAULT_PUPPET_AGENT_VERSION